compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -480,7 +482,22 @@ object Contexts {
 
     /** A new context that summarizes an import statement */
     def importContext(imp: Import[?], sym: Symbol): FreshContext =
-       fresh.setImportInfo(ImportInfo(sym, imp.selectors, imp.expr))
+      fresh.setImportInfo(ImportInfo(sym, imp.selectors, imp.expr))
+
+    def scalaRelease: ScalaRelease =
+      val releaseName = base.settings.scalaRelease.value
+      if releaseName.nonEmpty then ScalaRelease.parse(releaseName).get else ScalaRelease.latest
+
+    def tastyVersion: TastyVersion =
+      import math.Ordered.orderingToOrdered
+      val latestRelease = ScalaRelease.latest
+      val specifiedRelease = scalaRelease
+      if ((specifiedRelease.majorVersion, specifiedRelease.minorVersion) < (latestRelease.majorVersion, latestRelease.majorVersion)) then
```

Comment:
`latestRelease.majorVersion` used twice.

Reply:
Good catch. Actually I spotted that later too but haven't pushed my changes yet.